kobo/conf.py

```diff
@@ -160,7 +160,7 @@ def _get_token(self, skip_newline=True):
         self._tok_name = token.tok_name.get(self._tok_number, None)
 
         if self._debug:
-            print "%2s %16s %s" % (self._tok_number, self._tok_name, self._tok_value.strip())
+            print("%2s %16s %s" % (self._tok_number, self._tok_name, self._tok_value.strip()))
```

Comment:
I'm fine with that if you add:
from __future__ import print_function
to preserve python2.6 functionality